tool/planet/start.go

```diff
@@ -1057,20 +1028,50 @@ func unitNames(units map[string]string) []string {
 	return out
 }
 
-func getStatus(c libcontainer.Container, unit string) (string, error) {
-	out, err := box.CombinedOutput(
-		c, box.ProcessConfig{
+func getStatus(b *box.Box, unit string) (status string, err error) {
+	out, err := b.CombinedOutput(
+		box.ProcessConfig{
 			User: "root",
 			Args: []string{
 				"/bin/systemctl", "is-active",
 				fmt.Sprintf("%v.service", unit),
-			}})
-	if err != nil {
-		return "", err
+			},
+		})
+	if err == nil {
+		return serviceActive, nil
 	}
-	return strings.TrimSpace(string(out)), nil
+	return strings.TrimSpace(string(out)), trace.Wrap(err)
 }
 
 func isRoot() bool {
 	return os.Geteuid() == 0
 }
+
+var masterUnits = []string{
```

Comment:
Probably not relevant to this PR but I wonder why masterUnits does not include kube-proxy and kube-kubelet, while nodeUnits does? 🤔